build-logic/settings.gradle.kts

```diff
@@ -23,26 +23,23 @@ pluginManagement {
 dependencyResolutionManagement {
     repositories {
         maven {
-            name = "Gradle libs"
-            url = uri("https://repo.gradle.org/gradle/libs")
-            mavenContent {
-                // This repository contains an older version which has been overwritten in Central
-                excludeModule("com.google.j2objc", "j2objc-annotations")
+            name = "Gradle Enterprise release candidates"
+            url = uri("https://repo.gradle.org/gradle/enterprise-libs-release-candidates")
+            content {
+                val rcAndMilestonesPattern = "\\d{1,2}?\\.\\d{1,2}?(\\.\\d{1,2}?)?-((rc-\\d{1,2}?)|(milestone-\\d{1,2}?))"
+                includeVersionByRegex("com.gradle", "gradle-enterprise-gradle-plugin", rcAndMilestonesPattern)
+                includeVersionByRegex("com.gradle.enterprise", "test-distribution-gradle-plugin", rcAndMilestonesPattern)
             }
         }
-        gradlePluginPortal()
         maven {
-            name = "Gradle snapshot libs"
-            url = uri("https://repo.gradle.org/gradle/libs-snapshots")
-            mavenContent {
-                // This repository contains an older version which has been overwritten in Central
-                excludeModule("com.google.j2objc", "j2objc-annotations")
+            name = "Gradle public repository"
+            url = uri("https://repo.gradle.org/gradle/public")
+            content {
+                includeModule("classycle", "classycle")
             }
         }
-        maven {
-            name = "ge-release-candidates"
-            url = uri("https://repo.gradle.org/gradle/enterprise-libs-release-candidates-local")
-        }
+        gradlePluginPortal()
+        mavenCentral()
```

Comment:
we may want to swap these because gradlePluginPortal also mirrors all of jcenter currently.

Reply:
I've swapped the repository declarations although we can't be sure we're not resolving something via gradlePluginPortal that we shouldn't without declaring content filters. Given that most plugins contain something like `gradle-plugin` or `plugin-gradle` it might be doable to cover all plugins via a few module regexes. We can do that in a follow up from my PoV.